collectfast/management/commands/collectstatic.py

```diff
@@ -119,7 +119,8 @@ def delete_file(self, path, prefixed_path, source_storage):
                 path, prefixed_path, source_storage)
         if not self.dry_run:
             self.log("Deleting '%s'" % path)
-            self.storage.delete(prefixed_path)
+            if self.storage.exists(prefixed_path):
```

Comment:
I believe this introduces an extra network call and should not be introduced.

Reply:
Without this, if the file has already been deleted on GCS, an exception will be thrown. It's not a requirement though. Perhaps I can catch the exception instead, otherwise I'll remove this code entirely

Reply:
Definitely prefer catching the exception :)

Reply:
I removed this logic for now; I'll do a separate P/R if necessary later